### PR TITLE
New detection and announcing 2 new detection

### PR DESCRIPTION
**Pull request by unknownhad**
New detection and announcing 2 new detection

### CODE REVIEW

File: content/waf/change-log/2022-03-07.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Previous Action </th> <th>
+New Action </th> </tr> </thead> <tbody> <tr> <td>
+Cloudflare Specials </td> <td>
+...3fccf643 </td> <td>
+100519 </td> <td>
+Magento - CVE:CVE-2022-24086 </td> <td>
+N/A </td> <td>
+Block </td> </tr> {{</table-wrap>}}
+
+</table></table-wrap>
```
Comment (pedrosousa) [inline content/waf/change-log/2022-03-07.md:25]:
```suggestion
Block </td> </tr> 
</tbody>
</table>{{</table-wrap>}}
```

File: content/waf/change-log/scheduled-changes.md [hunk 2]
```diff
@@ -17,12 +17,23 @@ Legacy Rule ID </th> <th>
...
-100519 </td> <td>
-Magento - CVE:CVE-2022-24086 </td> <td>
+N/A </td> <td>
+100520 </td> <td>
+Apache - CVE:CVE-2022-24112 </td> <td>
+N/A </td> <td>
+Block </td> </tr>
+
+</tbody>
+<tbody> <tr> <td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md:30]:
```suggestion
<tr><td>
```

**Comment by pedrosousa** [comment]
FYI: I will soon re-format all these HTML code blocks using `prettier`... There's an almost-ready PR with these changes:
https://github.com/cloudflare/cloudflare-docs/pull/3734

You can go ahead and merge this PR whenever it's ready, and I'll update the PR I mentioned with the re-formatted version of the table.

**Review by gpgabriel — APPROVED** [review]

File: content/waf/change-log/2022-03-07.md (hunk 1 above)
Comment (pedrosousa) [reply content/waf/change-log/2022-03-07.md:25]: Note: This issue was *not* resolved. Next time, please fix identified issues before merging.

File: content/waf/change-log/scheduled-changes.md (hunk 2 above)
Comment (pedrosousa) [reply content/waf/change-log/scheduled-changes.md:30]: Note: This issue was *not* resolved. Next time, please fix identified issues before merging.

**Pull request merged**